### PR TITLE
**Do not merge** ACM-7922

### PR DESCRIPTION
**Pull request by krishnanshruthi**

### CODE REVIEW

**Comment by krishnanshruthi** [comment]
Created PR by mistake, doesnt need merging as of now, hence closing it.
**Do not merge**

File: clusters/hosted_control_planes/hosted-bm-agents-ibmz.adoc [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
++
+----
+rd.neednet=1 console=ttysclp0         coreos.live.rootfs_url=<rootfs_url> ip=<IP for guest vm>::<nameserver>:255.255.255.0 nameserver=<nameserver> rd.<zfcp/dasd>=<storage> random.trust_cpu=on rd.luks.options=discard ignition.firstboot ignition.platform.id=metal console=tty1 console=ttyS1,115200n8 coreos.inst.persistent-kargs="console=tty1 console=ttyS1,115200n8" 
+----  
+
+. Punch 'initrd' ,  kernel images and paramfile to guest VM by running the following commands:
+
++
+----
+vmur pun -r -u m1321006 -N kernel.img $INSTALLERKERNELLOCATION/<image name>
```
Comment (Neeraj8418) [inline clusters/hosted_control_planes/hosted-bm-agents-ibmz.adoc:38]: vmur pun -r -u <guestVM> -N kernel.img $INSTALLERKERNELLOCATION/<image name>

File: clusters/hosted_control_planes/hosted-bm-agents-ibmz.adoc [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+vmur pun -r -u m1321006 -N generic.parm $PARMFILELOCATION/paramfilename
```
Comment (Neeraj8418) [inline clusters/hosted_control_planes/hosted-bm-agents-ibmz.adoc:39]: same as above

File: clusters/hosted_control_planes/configure_hosted_bm_ibmz.adoc [hunk 3]
```diff
@@ -0,0 +1,73 @@
+[#configuring-hosting-service-cluster-configure-ibmz]
+= Configuring the hosting cluster on IBM Z (Technology Preview)
```
Comment (v78singh) [inline clusters/hosted_control_planes/configure_hosted_bm_ibmz.adoc:2]: Configuring the hosting cluster on x86 bare metal for IBM Z compute nodes

File: clusters/hosted_control_planes/configure_hosted_bm_ibmz.adoc [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+Hosted control planes is a Technology Preview feature, so the related components are disabled by default. 
+
+On {product-title-short} 2.7, you can use the managed hub cluster, also known as the `local-cluster`, as the hosting cluster.
+
+*Important:* 
+
+- To provision hosted control planes on bare metal, you can use the Agent platform. The Agent platform uses the Central Infrastructure Management (CIM) service to add worker nodes to a hosted cluster. For an introduction to the CIM service, see link:https://github.com/openshift/assisted-service/blob/master/docs/hive-integration/kube-api-getting-started.md[Kube API - Getting Started Guide].
+
+- Each IBM Power and Z system host must be started with the PXE images that is provided by the CIM. After each host starts, it runs an Agent process to discover the details of the host and completes the installation. An Agent custom resource represents each host.
```
Comment (v78singh) [inline clusters/hosted_control_planes/configure_hosted_bm_ibmz.adoc:20]: Remove IBM Power and keep only IBM Z

File: clusters/hosted_control_planes/configure_hosted_bm_ibmz.adoc [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+- When you scale up a node pool, a machine is created. The Cluster API provider finds an Agent that is approved, is passing validations, is not currently in use, and meets the requirements that are specified in the node pool specification. You can monitor the installation of an Agent by checking its status and conditions.
+
+- When you scale down a node pool, Agents are unbound from the corresponding cluster. Before you re-use the clusters, you must boot the clusters using the PXE image to update the number of nodes.
+
+[#hosting-service-cluster-configure-prereq]
+== Prerequisites
+
+You must have the following prerequisites to configure a hosting cluster: 
+
+* You need the {mce} 2.2 and later installed on an {ocp-short} cluster. The {mce-short} is automatically installed when you install {product-title-short}. You can also install {mce-short} without {product-title-short} as an Operator from the {ocp-short} OperatorHub.
```
Comment (v78singh) [inline clusters/hosted_control_planes/configure_hosted_bm_ibmz.adoc:35]: Even though MCE 2.3 has multi-arch images, its better to mention MCE 2.4 which is from OCP 4.14.

Comment (v78singh) [inline clusters/hosted_control_planes/configure_hosted_bm_ibmz.adoc:35]: 2.4

File: clusters/hosted_control_planes/configure_hosted_bm_ibmz.adoc [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+*Note:* The management cluster is not the same thing as the _managed_ cluster. A managed cluster is a cluster that the hub cluster manages.
+
+You can enable a managed cluster to be a hosting cluster by using the `hypershift` add-on to deploy the HyperShift Operator on that cluster. Then, you can start to create the hosted cluster. 
+
+The {mce-short} 2.4 supports only the default `local-cluster`, which is a hub cluster that is managed, and the hub cluster as the hosting cluster.
+
+Hosted control planes is a Technology Preview feature, so the related components are disabled by default. 
+
+On {product-title-short} 2.7, you can use the managed hub cluster, also known as the `local-cluster`, as the hosting cluster.
```
Comment (v78singh) [inline clusters/hosted_control_planes/configure_hosted_bm_ibmz.adoc:14]: Remove this line

File: clusters/hosted_control_planes/configure_hosted_bm_ibmz.adoc (hunk 5 above)
Comment (v78singh) [reply clusters/hosted_control_planes/configure_hosted_bm_ibmz.adoc:35]: use mce version 2.4 everywhere in doc

File: clusters/hosted_control_planes/configure_hosted_bm_ibmz.adoc [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+== IBM Z infrastructure requirements
+
+The Agent platform does not create any infrastructure, but it does have the following requirements for infrastructure:
+
+* Agents: An _Agent_ represents a host that is booted with a discovery image or PXE image and is ready to be provisioned as an {ocp-short} node.
+
+* DNS: The API and Ingress endpoints must be routable.
+
+[#configuring-hcp-bare-metal]
+== Configuring hosted control planes on bare metal
```
Comment (v78singh) [inline clusters/hosted_control_planes/configure_hosted_bm_ibmz.adoc:59]: Configuring hosted control planes on x86 bare metal for IBM Z compute nodes

File: clusters/hosted_control_planes/configure_hosted_bm_ibmz.adoc [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+
+After you meet the prerequisites, complete the following steps to configure hosted control planes on bare metal:
+
+. xref:../hosted_control_planes/hosted_bare_metal_dns.adoc#configure-dns-bm[Configure DNS on bare metal]
```
Comment (v78singh) [inline clusters/hosted_control_planes/configure_hosted_bm_ibmz.adoc:63]: Configure DNS for Hosted Control Plane

**Pull request closed without merging**